### PR TITLE
Update vendoring for logrus and gojsonschema changes

### PR DESCRIPTION
**Pull request by cdrage**
Some vendoring is blocking other PR's due to changes to logrus (Sirupsen                                                                                                                                                                                                          
vs sirupsen for lower-case) as well as a non-versioned gojsonschema in                                                                                                                                                                                                            
glide.yaml                                                                                                                                                                                                                                                                        
                                                                                                                                                                                                                                                                                  
This updates glide.yaml to reflect the upper-case as well as lower-case                                                                                                                                                                                                           
versions of logrus as well as adds a versionized gojsonschema in order                                                                                                                                                                                                            
to get rid of the following error:                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                  
```sh                                                                                                                                                                                                                                                                             
▶ make bin                                                                                                                                                                                                                                                                        
go build -ldflags="-w -X github.com/kubernetes/kompose/cmd.GITCOMMIT=0e56b7d" -o kompose main.go                                                                                                                                                                                  
# github.com/kubernetes/kompose/vendor/github.com/docker/cli/cli/compose/schema                                                                                                                                                                                                   
vendor/github.com/docker/cli/cli/compose/schema/schema.go:34: cannot use portsFormatChecker literal (type portsFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                                  
        portsFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                         
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
vendor/github.com/docker/cli/cli/compose/schema/schema.go:35: cannot use portsFormatChecker literal (type portsFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                                  
        portsFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                         
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
vendor/github.com/docker/cli/cli/compose/schema/schema.go:36: cannot use durationFormatChecker literal (type durationFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                            
        durationFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                      
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
# github.com/kubernetes/kompose/vendor/github.com/docker/libcompose/config                                                                                                                                                                                                        
vendor/github.com/docker/libcompose/config/schema_helpers.go:60: cannot use environmentFormatChecker literal (type environmentFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                   
        environmentFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                   
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
vendor/github.com/docker/libcompose/config/schema_helpers.go:61: cannot use portsFormatChecker literal (type portsFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                               
        portsFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                         
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
vendor/github.com/docker/libcompose/config/schema_helpers.go:62: cannot use portsFormatChecker literal (type portsFormatChecker) as type gojsonschema.FormatChecker in argument to gojsonschema.FormatCheckers.Add:                                                               
        portsFormatChecker does not implement gojsonschema.FormatChecker (wrong type for IsFormat method)                                                                                                                                                                         
                have IsFormat(string) bool                                                                                                                                                                                                                                        
                want IsFormat(interface {}) bool                                                                                                                                                                                                                                  
^CMakefile:29: recipe for target 'bin' failed                                                                                                                                                                                                                                     
make: *** [bin] Interrupt                                                                                                                                                                                                                                                         
```

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -4,8 +4,16 @@ licence: Apache-2.0
 
 import:
 
+- package: github.com/sirupsen/logrus
+  version: master
```
Comment (kadel) [inline glide.yaml]: It would be better to pin this to some specific version.

File: glide.yaml [hunk 2]
```diff
@@ -4,8 +4,16 @@ licence: Apache-2.0
 
 import:
 
+- package: github.com/sirupsen/logrus
+  version: 1.0.3
+
 - package: github.com/Sirupsen/logrus
-  version: v0.11.5
+  repo: git@github.com:/sirupsen/logrus
```
Comment (kadel) [inline glide.yaml]: Is there another way to resolve this other than vendoring this twice?

Comment (cdrage) [reply glide.yaml]: Nope :-1: https://github.com/sirupsen/logrus/issues/570

**Comment by surajnarwade** [comment]
@cdrage , bit confused here but I am not able to see `sirupsen/logrus` in vendor directory (instead there `Sirupsen/logrus`)

**Comment by cdrage** [comment]
@surajnarwade There's conflicts with docker/cli updating to using `sirupsen/logrus` instead of `Sirupsen/logrus`.

Yes, there is no folder for it, Glide handles it.

**Comment by surajnarwade** [comment]
LGTM

**Pull request merged**